components/link/README.md

```diff
@@ -0,0 +1,103 @@
+---
+description: Links lead users to a different page or further information.
+title: Links
+status: Draft
+author: orinevares
+---
+
+![Status](https://img.shields.io/badge/Component-Draft-orange.svg)
+
+# Links
+
+Links lead users to a different page or further information.
+
+### Required
+
+## Last Updated:
+
+## Visual
+Link Visual:
+![Link](https://github.com/bcgov/design-system/blob/master/components/link/images/link.png?raw=true)
+
+## Requirements
+* This component must appear on all public-facing online B.C. Government content and services. Any feedback should be directed to the feedback forum.
+
+## Don't Use This when:
+* Users need to take an action on a page such as log-in or submit, use a primary button instead.
+
+## Design Protocol
+* If your link is at the end of a paragraph or sentence make sure the linked text does not include the full stop.
+
+## Behaviour
+
+1.	Link starts out as government link colour, #1a5a96.
+2.	On hover link removes underline and turns to blue.
+
+## Accessibility
+This checkbox has been built according to [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards and all government services should strive to meet this level.  This component successfully includes the following accessibility features:
+
+### Screenreaders
+* Proper ALT labels are included
+* Language tag has been set to english
+
+### Colour Contrast
+* Contrast ratios are at least 7:1 for normal text and 4.5:1 for large text
+* Users can identify the link without relying on colour alone
+
+### Tab Navigation
+* Supports tab navigation for use without a mouse
+
+### Learn More
+* Contacts
+* Online Tools
+* Online Resources
+* Gov Resources
```

Comment:
If these Learn More resources end up being links, we should make sure that we follow our own advice about how to write links descriptively